service/awsconfig/v9/resource/ebsvolume/delete.go

```diff
@@ -16,21 +16,40 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 		return microerror.Mask(err)
 	}
 
-	if deleteInput != nil && len(deleteInput.VolumeIDs) > 0 {
-		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("deleting %d ebs volumes", len(deleteInput.VolumeIDs)))
+	if deleteInput != nil && len(deleteInput.Volumes) > 0 {
+		r.logger.LogCtx(ctx, "level", "info", "message", fmt.Sprintf("deleting %d ebs volumes", len(deleteInput.Volumes)))
```

Comment:
I upgraded the logging to info as well as adding debug logging for detaching and deleting each volume.